sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -259,6 +273,18 @@ const argv = yargs(hideBin(process.argv))
     type: "number",
     default: 100,
   })
+  .option("with-latency", {
```

Comment:
These options are mainly for internal testing and ideally we shouldn't expose them in this simple searcher.
Can we have another script and another class like `ExtendedSearcherLimo` that inherits the `SimpleSearcherLimo` and just overrides the opportunityHandler? We can add the latency there and change the bid each time before calling the generateBid function. Therefore the base script remains the same.